examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -16,25 +16,26 @@ enum CardDemoType {
 }
 
 class TravelDestination {
-  const TravelDestination({
-    this.assetName,
-    this.assetPackage,
-    this.title,
-    this.description,
+  TravelDestination({
+    @required this.assetName,
+    @required this.assetPackage,
+    @required this.title,
+    @required this.description,
     this.type = CardDemoType.standard,
-  });
+  }) : assert(assetName != null),
+       assert(assetPackage != null),
+       assert(title != null),
+       assert(description != null && description?.length == 3);
```

Comment:
assert(description != null && description.length == 3)

This constructor should be const

Reply:
It looks like I am unable to check `description.length` and use a `const` constructor. I get the error `Initializer expressions in constant constructors must be constant`.

I'll remove the `?` however.

Reply:
On second thought, I changed description from an array of 3 elements to 3 separate string properties as that made more logical sense. I'm now able to use a const constructor too. So done!